quad.go

```diff
@@ -0,0 +1,159 @@
+// Copyright ©2015 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package quad provides numeric integration of single variable functions by quadrature.
+//
+package quad
```

Comment:
I doubt there is a package just for quadrilaterals. I have had a mesh.Quad type, but that doesn't clash. In general one is supposed to keep package names short, but quadrature.Fixed may be more informative? On the other hand, quad is the function name in matlab and python. I can go either way.